src/main/java/com/synopsys/integration/alert/provider/blackduck/collector/BlackDuckCollector.java

```diff
@@ -85,6 +89,28 @@ public BlackDuckCollector(final JsonExtractor jsonExtractor, final List<MessageC
         return Optional.empty();
     }
 
+    public Optional<VersionBomComponentView> getBomComponentView(final String bomComponentUrl) {
+        if (getBucketService().isPresent()) {
+            try {
+                getBucketService().get().addToTheBucket(getBlackDuckBucket(), bomComponentUrl, VersionBomComponentView.class);
+                return Optional.ofNullable(getBlackDuckBucket().get(bomComponentUrl, VersionBomComponentView.class));
+            } catch (final IntegrationException ie) {
+                logger.error("Error retrieving bom component/", ie);
+                return Optional.empty();
+            }
+        }
+        return Optional.empty();
+    }
+
+    public List<LinkableItem> getLicenseLinkableItems(final VersionBomComponentView bomComponentView) {
+        return bomComponentView.getLicenses().stream().map(licenseView -> {
+            LinkableItem item = new LinkableItem(BlackDuckContent.LABEL_COMPONENT_LICENSE, licenseView.getLicenseDisplay(), licenseView.getLicense());
+            item.setSummarizable(true);
```

Comment:
Is license something relevant to a summary view?

Reply:
no it actually isn't I will remove that. We don't summarize by license here.